extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -161,7 +160,8 @@ public static JavaSettings getInstance() {
                 getBooleanValue(host, "no-custom-headers", false),
                 getBooleanValue(host, "include-read-only-in-constructor-args", false),
                 // setting the default as true as the Java design guideline recommends using String for URLs.
-                getBooleanValue(host, "url-as-string", true)
+                getBooleanValue(host, "url-as-string", true),
+                host.getValue(new TypeReference<Map<String, String>>() {}.getType(), "custom-package-info-descriptions")
```

Comment:
@alzimmermsft - can customization be used to update the package info javadocs?

We should solve the docs customization issue holistically. As of now, using customization is the recommended way.

Reply:
It can be done via code customizations but that can get confusing when a library uses multiple Swaggers as part of code generation.

At the same time, I do agree that this is more a temporary solution than a long-term solution. But, this is better than the provided hardcoded `package-info.java` javadoc descriptions and is a low cost add.